### PR TITLE
Keyboard tests

### PR DESCRIPTION
**Pull request by stechkev**
https://github.com/willowtreeapps/vocable-ios/issues/291

closes 291

*Add the ticket number to above*

# Description of Work
Adding basic keyboard tests

## Notes to Test (Optional)
Notes

---

### CODE REVIEW

File: VocableUITests/Screens/KeyboardScreen.swift [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+//  Copyright © 2020 WillowTree. All rights reserved.
+//
+
+import XCTest
+
+class KeyboardScreen {
+    let app = XCUIApplication()
+    static let keyboardTextView = XCUIApplication().collectionViews.children(matching: .cell).element(boundBy: 0).children(matching: .other).element(boundBy: 1)
+    static let mySayingsSaveButton = XCUIApplication().collectionViews.children(matching: .cell).element(boundBy: 1).children(matching: .staticText).element
+    static let returnToMainScreenButton = XCUIApplication().collectionViews.children(matching: .cell).element(boundBy: 2).children(matching: .staticText).element
```
Comment (nerdsosky) [inline VocableUITests/Screens/KeyboardScreen.swift:15]: yeah we will need some accessibility id's soon, these are a bit verbose

File: VocableUITests/Tests/KeyboardScreenTests.swift [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+class VocableUITests: BaseTest {
+    let app = XCUIApplication()
+    let keyboardScreen = KeyboardScreen()
+    let mainScreen = MainScreen()
+    let testPhrase = "Test"
+
+    override func setUp() {
+        continueAfterFailure = false
+        app.launch()
```
Comment (nerdsosky) [inline VocableUITests/Tests/KeyboardScreenTests.swift]: This functionality can probably go in BaseTest since we will be launching the app everytime

**Review by nerdsosky — APPROVED** [review]
Looks good, some minor comments

File: VocableUITests/Screens/KeyboardScreen.swift (hunk 1 above)
Comment (stechkev) [reply VocableUITests/Screens/KeyboardScreen.swift:15]: I can't wait...

**Pull request merged**